ui/src/app/pages/admin/user/admin-user-table.tsx

```diff
@@ -288,56 +250,50 @@ export const AdminUserTable = withUserProfile()(
                 paginator={true}
                 rows={50}
                 scrollable
-                sortField={'firstSignInTimestamp'}
                 style={styles.tableStyle}
               >
                 <Column
                   field='name'
                   bodyStyle={{ ...styles.colStyle }}
-                  filterField={'nameText'}
-                  filterMatchMode={'contains'}
+                  filterField='nameText'
+                  filterMatchMode='contains'
                   frozen={true}
                   header='Name'
                   headerStyle={{ ...styles.colStyle, width: '200px' }}
                   sortable={true}
-                  sortField={'nameText'}
-                />
-                <Column
-                  field='institutionName'
-                  bodyStyle={{ ...styles.colStyle }}
-                  header='Institution'
-                  headerStyle={{ ...styles.colStyle, width: '180px' }}
-                  sortable={true}
+                  sortField='nameText'
                 />
                 <Column
                   field='username'
                   bodyStyle={{ ...styles.colStyle }}
-                  header='User Name'
```

Comment:
IMO: This text was kinda ambiguous vs. "Name", and "Username" is a clearer distinction